pyaedt/edb_grpc/core/edb_data/edbvalue.py

```diff
@@ -0,0 +1,53 @@
+class EdbValue:
+    """Class defining Edb Value properties."""
+
+    def __init__(self, edb_obj):
+        self._edb_obj = edb_obj
+
+    @property
+    def value(self):
+        """Variable Value Object.
+
+        Returns
+        -------
+        Edb Object
```

Comment:
```suggestion
        EDB object.
```
What's here should be the indented description. The data object link should be in the first line.